tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```

Comment:
Add test cases:
 - multiple patients
 - more varying data closer to the real data (values up to 100)
 - test case that the correct error is raised if the data is empty

Reply:
Partially Implemented through latest commit (bcc8a02). Test cases for multiple patients and correct error has not yet been implemented due to some issues that came up, but will try to do so in a future commit!